gensim/models/poincare.py

```diff
@@ -53,7 +53,7 @@
 from scipy.stats import spearmanr
 from six import string_types
 from six.moves import zip, range
-from smart_open import smart_open
+from smart_open import open
```

Comment:
Why are we using two separate methods to open files?

1. smart_open.open
2. utils.open

?

Reply:
AFAICS `utils.open` is an alias of `smart_open.open` from that module.

I agree there's no point using that: just use the local `from smart_open import open` version.

@itayB can you please fix that too, while we're cleaning up?

Reply:
fixed - removed this import and change other usages to `utils.open`

Reply:
IMO `better use the local "from smart_open import open" version, consistently`.
But the other way around works too, I guess :)

Reply:
@piskvorky I changed everything to work with `utils.open` - should be consistented now